context_test.go

```diff
@@ -9,7 +9,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/couchbaselabs/go.assert"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
`go-blip` doesn't use testify anywhere else, so it seems like a bad idea to introduce a new dependency to the project for one new test.
The testify import breaks existing assertions (parameters are in a different order). I think you can use `go.assert`'s .Equals() instead of testify's `Equal()` in your new test once you've swapped the parameter ordering.

As much as I hate to say it, you can replace the `requrie.NoError()`s with panics for now, just to keep the new test consistent with the others.

Reply:
I hate this very much but fine 😛